docs/src/inputs/segmentedControls/segmentedControls.jsx

```diff
@@ -171,18 +171,19 @@ export default class CollectionsSegmentedControls extends React.Component {
             <Main page="headers">
                 <Main.Content>
                     <Card>
-                        <Header size="large">Props</Header>
+                        <Typography size="large">Props</Typography>
 
                         <TableProps props={props} />
                     </Card>
 
                     {/* Segmented Controls */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub={true}>
+                    <Typography size="large" style={{ marginTop: '55px' }} sub={true}>
```

Comment:
I imagine we no longer need `sub={true}` here, right?

Reply:
MMM, let me see if I can do a find a replace on this.

Reply:
Fixed.

Reply:
Looks good.  Resolving.